### PR TITLE
Lecture16

### PR DESCRIPTION
**Pull request by Sue838**

### CODE REVIEW

File: sanity_test.py [hunk 1]
```diff
@@ -1,57 +1,7 @@
...
-
-def get_sensor_reading():
-    payload = make_valid_payload(method="get_reading")
-    sensor_response = send_post(**payload)
-    sensor_reading = sensor_response.get("result", {})
-    return sensor_reading
-
-
-def test_sanity():
+def test_sanity(get_sensor_info, get_sensor_reading, get_sensor_methods, set_sensor_name, set_sensor_reading_interval, reset_sensor_to_factory, update_sensor_firmware, sensor_reboot):
```
Comment (ddoroshevskyi) [inline sanity_test.py]: This test needs only `get_sensor_info` and `get_sensor_reading`, so the rest of the fixtures need to be removed from it.

**Review by ddoroshevskyi — CHANGES_REQUESTED** [review]

File: sanity_test.py (hunk 1 above)
Comment (Sue838) [reply sanity_test.py]: I have made the changes in sanity_test.py. I gained to save changes in the same commit, but it seams I have merged the changes in main branch.
$ git log
commit a7c4b77ad9c4dbc65e0130d32fd7e79858c11042 (HEAD -> main)
Merge: 8edc3b1 6b89156
Author: Sue838 <suelight838@gmail.com>
Date:   Tue Apr 2 21:02:11 2024 +0200

    Merge branch 'main' of https://github.com/Sue838/fw_at-iot

Comment (Sue838) [reply sanity_test.py]: Before push the changes I've checked git status and I don't have any ideas why I have got following modifications:
$ git status
On branch main
Your branch is ahead of 'origin/main' by 3 commits.
  (use "git push" to publish your local commits)

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
        modified:   venv/Lib/site-packages/_distutils_hack/__pycache__/__init__.cpython-311.pyc

Comment (ddoroshevskyi) [reply sanity_test.py]: We'll go over how to fix issues like this in today's lecture. After that you should be able to deal with conflicts you have in this PR.

**Pull request merged**